tenant_schemas/test/cases.py

```diff
@@ -1,28 +1,31 @@
 from django.core.management import call_command
 from django.db import connection
-from django.test import TestCase
+from django.test import TransactionTestCase
```

Comment:
Parallel migrations tests fails because django.test.TestCase doesn't handle connection.close properly (because of performance reasons, it applies migrations and then does a rollback). For that reason I rewrite this to django.test.TransactionTestCase (please read more here https://docs.djangoproject.com/en/1.10/topics/testing/tools/#django.test.TransactionTestCase where it explicitly states about TestCase using commit and rollback).